applications/TREX_Network.md

```diff
@@ -0,0 +1,284 @@
+be recorded on our blockchain, thus, becoming immutable. The encrypted data would be decrypted and released in the future after the event occurs (e. g. market closed and the final price of an asset was revealed). Since the on-chain data is 
+immutable, the oracles cannot delete their wrong suggestions or predictions. All of their previous data would generate a reputation for themselves.
+
+Third-party on-chain data services, which disassemble the block data into extrinsic and analyze encrypted predictions or strategies, could generate ranks or recommendations based on specific metrics (e. g. return of investment). The end users may get early access to an oracle's data by paying a fee. Thus, end users can use the predictions and investment strategy, which were created by the oracles, for their own interests. 
+
+Since the TREX software is fully open source and free to use, the users only pay fees to use the infrastructure. The tokens on the TREX network could be used to pay insertion fees for writing dTRE data on the blockchain. The data fee is comprised of a base fee and a data fee per byte. Since there is an upper limit on block size, the data fee is dynamically adjusted based on network usage.
+
+New tokens were created by miners on the TREX network and exchanged to end users.
+
+- An overview of the technology stack to be used
+
+Our project is exclusively developed with Polkadot substrate ecosystems.
+
+  1. Our [TREX node software](https://github.com/NexTokenTech/TREX) is based on the latest Substrate framework, which is under internal network test and is to be released as open source software.
+  2. Our [TREX blockchain index engine](https://github.com/NexTokenTech/trex-archive) is based on the Substrate Archive. It releases timed-release encrypted data and archives dTRE data into a PostgreSQL database.
+  3. For applications on the TREX network, we provide solutions including an [API library](https://github.com/NexTokenTech/trex-subxt) to our business partners who may use it to insert dTRE data on the TREX network and retrieve data with the TREX blockchain index engine.
+  4. The DevOps tool uses [Terraform](https://www.terraform.io) to automate the deployment of the TREX infrastructure with other backend services on cloud computing platforms so that our targeted clients can easily integrate the dTRE technology with their existing services.
+
+- Documentation of core components, protocols, architecture, etc. to be deployed
+
+Our white paper is our core documentation.
+
+<iframe src="https://onedrive.live.com/embed?cid=BEF98A7CB59E1060&resid=BEF98A7CB59E1060%21106391&authkey=AGFFIH66aCaJ9Yw&em=2" width="800" height="400" frameborder="0" scrolling="no"></iframe>
```

Comment:
`iframe`s are [not supported](https://github.github.com/gfm/#disallowed-raw-html-extension-) by GitHub markdown, could you remove it and maybe replace it with a normal link?